### PR TITLE
fix: I915205 - Include angus-mail runtime for jakarata.mail-api

### PR DESCRIPTION
**Pull request by raghavendran-gopalakrishnan**
https://internal.almoctane.com/ui/?p=131002/6001#/entity-navigation?entityType=work_item&id=915205

### CODE REVIEW

**Comment by andyreidz** [comment]
@rorytorneymf Sorry I can't get to this today, could you review?

**Comment by rorytorneymf** [comment]
Is this a new convention we are using for PR titles? We normally just use:

I915205: Include angus-mail runtime for jakarata.mail-api

File: autoscale-email-alert-dispatcher/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker [hunk 1]
```diff
@@ -0,0 +1 @@
+mock-maker-inline
```
Comment (rorytorneymf) [inline autoscale-email-alert-dispatcher/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker]: Does this file need a new line?

File: autoscale-email-alert-dispatcher/src/test/java/com/github/autoscaler/email/alert/dispatcher/EmailDispatcherTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        when(emailDispatcherConfiguration.getEmailAddressTo()).thenReturn("rg@ot.com");
+        when(emailDispatcherConfiguration.getHost()).thenReturn("dummyHost");
+        when(emailDispatcherConfiguration.getUsername()).thenReturn("dummyUserName");
+        when(emailDispatcherConfiguration.getPassword()).thenReturn(("dummyPassword"));
+        try {
+            EmailDispatcher emailDispatcher = new EmailDispatcher(emailDispatcherConfiguration);
+            emailDispatcher.dispatch("hello, this is a test email");
+        } catch (Exception e) {
+            Assert.fail("Error while trying to dispatch mail.");
+        }
```
Comment (rorytorneymf) [inline autoscale-email-alert-dispatcher/src/test/java/com/github/autoscaler/email/alert/dispatcher/EmailDispatcherTest.java]:
```suggestion
        } catch (final Exception e) {
            Assert.fail("Error while trying to dispatch mail: " + e);
        }
```

**Review by rorytorneymf — APPROVED** [review]
Just a couple of small comments, thanks

**Comment by raghavendran-gopalakrishnan** [comment]
> Is this a new convention we are using for PR titles? We normally just use:
> 
> I915205: Include angus-mail runtime for jakarata.mail-api

Yes just thought of incorporating https://www.conventionalcommits.org/en/v1.0.0/

**Comment by rorytorneymf** [comment]
> > Is this a new convention we are using for PR titles? We normally just use:
> > I915205: Include angus-mail runtime for jakarata.mail-api
> 
> Yes just thought of incorporating https://www.conventionalcommits.org/en/v1.0.0/

Ok no worries, think its maybe better to stay consistent though with what everyone in the rest of the team is using. 

We should maybe discuss if everyone in the team should be using this new convention @dermot-hardy?

**Comment by raghavendran-gopalakrishnan** [comment]
> > > Is this a new convention we are using for PR titles? We normally just use:
> > > I915205: Include angus-mail runtime for jakarata.mail-api
> > 
> > 
> > Yes just thought of incorporating https://www.conventionalcommits.org/en/v1.0.0/
> 
> Ok no worries, think its maybe better to stay consistent though with what everyone in the rest of the team is using.
> 
> We should maybe discuss if everyone in the team should be using this new convention @dermot-hardy?

I don't see any conventions used till now in Fusion, it has always been < TicketID > : < Title >.
Interestingly, UX-Aspects team follow this convention. It looks good. 
https://github.houston.softwaregrp.net/Inner-Source-Incubating/ux-aspects-universal/commits/master

**Pull request merged**